src/includes/getting-started-install/java.mdx

```diff
@@ -13,3 +11,22 @@ dependencies {
     implementation 'io.sentry:sentry:3.1.0'
 }
 ```
+
+```xml {tabTitle:Maven}
+<dependency>
+    <groupId>io.sentry</groupId>
+    <artifactId>sentry</artifactId>
+    <version>3.1.0</version>
+</dependency>
+```
+
+```scala {tabTitle:SBT}
+libraryDependencies += "io.sentry" % "sentry" % "3.1.0"
+```
+<Note>
+<markdown>
+
+We also support [manual usage](/platforms/java/usage/).
+
+</markdown>
+</Note>
```

Comment:
Given that this page is covering the manual setup maybe we can drop this line?


```suggestion
```

Reply:
Sure.